README.md

```diff
@@ -9,6 +9,8 @@ backplane-cli is a CLI tool to interact with [backplane api](https://github.com/
 
 ## Installation
 
+Go should be installed in your local system with version >= 1.19 
```

Comment:
I think maybe it is better to request the local go version to match with what the project is using.

Reply:
Ok, I'll keep it to only 1.19 then